pageserver/src/tenant.rs

```diff
@@ -721,6 +785,7 @@ impl Tenant {
                     }
                 }
 
+                let attach_timer = TENANT.attach.start_timer();
                 match tenant_clone.attach(preload, &ctx).await {
```

Comment:
nit: should measure just the attach, not including the activation.
I think we have an `Ext` trait for metrics histograms somewhere.

Also, follow-up: get rid of the unused second argument to `.activate()` below.

Reply:
bc24bf223